first English files

ungesundes-halbwissen opened the pull request:
syncing the project.

Not yet done /started:

Story
Voices

ungesundes-halbwissen committed:
first intro graphics

also: test commit to check if everything is good to go

ungesundes-halbwissen committed:
englisch tga graphics

Witches voice may be a placeholder

ungesundes-halbwissen committed:
first set of English texts

ungesundes-halbwissen committed:
Graphics cleanup

FlorianBorn71 commented on Data/SourceAssets/Localization/English/StartMsg.txt:
```diff
@@ -1,10 +1,10 @@
-           F1 : Hauptmenue
-           F2 : Rucksack
+         1..9 : Choose weapon
+           F1 : Main menue
```
typo: menu

FlorianBorn71 commented on Data/SourceAssets/Localization/English/StartMsg.txt:
```diff
@@ -1,10 +1,10 @@
-           F2 : Rucksack
+         1..9 : Choose weapon
+           F1 : Main menue
+           F2 : Backpack
```
suggestion: 'Inventory' instead of backpack?

FlorianBorn71 commented on Data/SourceAssets/Localization/English/Items.txt:
```diff
@@ -127,27 +127,27 @@
+(One stone costs  = Gold coins
+(The medicine costs = Gold coins
+(The blade costs  = Gold coins
+(The sword costs    = Gold coins
```
why the whitespaces before the = ? Since we are packing the files again, the lengths do not have to match, unless for on-screen formatting reasons.

ungesundes-halbwissen replied:
No clue, where these came from. Thanks for all the corrections. Applied as suggested

FlorianBorn71 commented on Data/SourceAssets/Localization/English/IngameTrainingInstructions.txt:
```diff
@@ -1,14 +1,14 @@
+Press F2 to bring up the inventory \and drag the healing potion unto \the icon �knife and fork.
+Pick up all weapons in this room.\You will need them...
+Climb the ladder using \the JUMP hotkey.
+Jump in the water.
```
Jump into

FlorianBorn71 commented on Data/SourceAssets/Localization/English/IngameTrainingInstructions.txt:
```diff
@@ -1,14 +1,14 @@
-Spring ins Wasser.
-Ziehe unter Wasser den Hebel\und tauche zum Ausgang.
+Enter the room right in front of you.
+Stand right in front of the door.\Press �Return
```
Not correct in the original version, but please fix: Full stop after "Press Return"

FlorianBorn71 commented on Data/SourceAssets/Localization/English/EndMsg.txt:
```diff
@@ -1,15 +1,15 @@
-DESPERABIS ist freeware, d.h. die Nutzung und Verbreitung ist erlaubt.
-Wenn Dir das Spiel gef�llt, bitte ich um email-Nachricht an:
+DESPERABIS is freeware, and thus, free use and distribution is permitted.
```
fyi, none of the links (or email) still works, so I'm considering to re-link to github. But we can do this later.

ungesundes-halbwissen replied:
noted

FlorianBorn71 reviewed (changes requested):
some minor change requests

ungesundes-halbwissen committed:
Update IngameTrainingInstructions.txt

FlorianBorn71 commented on Data/SourceAssets/Localization/English/IngameTrainingInstructions.txt:
```diff
@@ -1,14 +1,14 @@
 Enter the room right in front of you.
-Stand right in front of the door.\Press �Return
+Stand right in front of the door.\Press ûReturn.
```
The diff shows that the special character before the "Return" has changed. I assume this happened by re-saving the file?
I suppose that character changed the text color, so maybe that does not work anymore?

ungesundes-halbwissen replied:
True, Github changed the encoding to UTF-8. Checking in from Source again

FlorianBorn71 replied:
Is there any github setting we should change for this repo?

ungesundes-halbwissen replied:
I am not aware of any setting that would allow to change the default encoding of Githubs Web-Editor. Google comes up empty as well. (same question but no answers). As we are now aware of the fact, it shouldn't be a big thing.

FlorianBorn71 replied:
@ungesundes-halbwissen , I changed the encoding rule for .txt in the .gitattributes - I hope this will prevent it from happening again

ungesundes-halbwissen committed:
Update StartMsg.txt

FlorianBorn71 commented on Data/SourceAssets/Localization/English/IngameTrainingInstructions.txt:
```diff
@@ -1,14 +1,14 @@
 Now take the healing potion\to restore your health.
-Press F2 to bring up the inventory \and drag the healing potion unto \the icon �knife and fork.
+Press F2 to bring up the inventory \and drag the healing potion unto \the icon ûknife and fork.
 Pick up all weapons in this room.\You will need them...
```
Same here, character before 'knife' changed

ungesundes-halbwissen committed:
Update Items.txt

ungesundes-halbwissen committed:
Update IngameTrainingInstructions.txt

Learnd something: resolving change requests online changes encoding from ANSI to UTF-8

ungesundes-halbwissen committed:
Merge branch 'FlorianBorn71:main' into main

ungesundes-halbwissen committed:
Merge branch 'FlorianBorn71:main' into main

ungesundes-halbwissen committed:
Update .gitattributes

text files would still be reencoded to UTF-8. Trying another environment variable

ungesundes-halbwissen committed:
Update .gitattributes

ungesundes-halbwissen committed:
Four typos in original text

ungesundes-halbwissen committed:
Merge branch 'main' of https://github.com/ungesundes-halbwissen/Desperabis

ungesundes-halbwissen committed:
tidying up

ungesundes-halbwissen committed:
Update .gitattributes

ungesundes-halbwissen committed:
tidying up 2
